docs/articles/expensify-classic/getting-started/Invite-Members.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Invite Members
+description: Learn how add your employees to submit expenses in Expensify
+---
+# Overview
+
+To invite your employees in Expensify, simply add them as members to your Workspace.
+
+# How to Invite members to Expensify
+
+## Inviting Members Manually
+
+Navigate to **Settings > Workspace > Group > *Workspace Name* > People** - then click Invite and enter the invitee's email address.
+
+Indicate whether you want them to be an Employee, Admin, or Auditor on the Workspace.
+
+If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.
```

Comment:
```suggestion
If you are utilizing the Advanced Approval feature and the invitee is an approver, you can use the "approves to" field to specify to whom they approve and forward reports to for additional approval.
```

Reply:
@NicMendonca I assume this sentence is about the "Approves to" field on the invite modal, is that correct? If yes, I suggest updating the sentence to the above suggestion